source/dynamical_systems/README.md

```diff
@@ -208,4 +217,115 @@ while (...) {
 
 ## Circular
 
-TODO
\ No newline at end of file
+The **Circular** DS is a limit cycle that rotates around a center point in an elliptical orbit,
+converging to a desired radius on a plane.
+ 
+This DS is defined only for the `CartesianState` type. In addition, it only acts in linear space,
+determining a linear velocity for a given position. It does not produce any angular velocity.
+
+The Circular DS can be constructed with a `CartesianState` state and radius as an argument;
+the state position defines the center of the limit cycle, while the state orientation defines the inclination
+of the limit cycle plane. The radius has a default value of 1.
+
+The internal representation of the limit cycle is a `state_representation::Ellipsoid` type. When a state is used
+in the constructor, the limit cycle has a constant radius.
+If an elliptical limit cycle is desired, the DS can be constructed directly from an `Ellipsoid` type.
+
+```c++
+// construct the circular DS limit cycle using a CartesianState center
+state_representation::CartesianState center("center");
+
+// default constructor (radius = 1)
+DynamicalSystems::Circular circularDS1(center);
+
+// constructor with radius
+double radius = 2.0;
+DynamicalSystems::Circular circularDS2(center, radius);
+
+// construct the circular DS limit cycle directly using an Ellipsoid type
+state_representation::Ellipsoid ellipse("limit_cycle");
+ellipse.set_center_state(center);
+ellipse.set_axis_lengths({radius, 2 * radius});
+DynamicalSystems::Circular ellipticalDS(ellipse);
+```
+
+### Configuring the Circular DS
+
+The Circular DS has the following core parameters:
+
+- **limit_cycle**; the `Ellipsoid` object defining the limit cycle center, shape and inclination 
+- **planar_gain**; the proportional gain acting in the local plane towards the limit cycle radius
+- **normal_gain**; the proportional gain acting towards the local plane
+- **circular_velocity**; the expected angular orbital velocity around the local origin
```

Comment:
I still have the same question about the velocity.. How do you know in what sense its gonna turn? Right hand rule?